api/src/main/java/com/datastrato/gravitino/rel/types/Types.java

```diff
@@ -923,4 +923,12 @@ public int hashCode() {
       return Arrays.hashCode(types);
     }
   }
+
+  /**
+   * @param dataType The data type to check.
+   * @return True if the given data type is allowed to be an auto-increment column.
+   */
+  public static boolean isAllowAutoIncrement(Type dataType) {
+    return dataType instanceof IntegerType || dataType instanceof LongType;
+  }
```

Comment:
Using 'isAllowAutoIncrement' as a member method in Types.java might not be appropriate. It is recommended to provide a Util class like `Strings` to implement this.